aci/data_source_aci_fvesg.go

```diff
@@ -0,0 +1,129 @@
+package aci
+
+
+import (
+	"fmt"
+	"github.com/ciscoecosystem/aci-go-client/client"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+func dataSourceAciEndpointSecurityGroup() *schema.Resource {
+	return &schema.Resource{
+
+		Read:   dataSourceAciEndpointSecurityGroupRead,
+
+		SchemaVersion: 1,
+
+		Schema: AppendBaseAttrSchema(map[string]*schema.Schema{
+			"application_profile_dn": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			
+			
+			"name": &schema.Schema{
+				Type: schema.TypeString,
+				Required: true,
+			},
+			
+			
+            
+			
+			"annotation": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+			},
```

Comment:
Annotation field is not required here, because I added annotation in base schema.